templates/preempt-rt/preempt-rt.jinja2

```diff
@@ -0,0 +1,30 @@
+- test:
+    timeout:
+      minutes: 10
+
+    definitions:
+    - from: inline
+      repository:
+        metadata:
+          format: Lava-Test Test Definition 1.0
+          name: preempt-rt-prereq
+          description: Pre-requisites for PREEMPT_RT
+        run:
+          steps:
+          - apt-get update && apt-get install -y procps rt-tests
+      name: preempt-rt-prereq
+      path: inline/preempt-rt-prereq.yaml
+
+    - repository: https://github.com/kernelci/test-definitions.git
+      from: git
+      revision: staging.kernelci.org
```

Comment:
```suggestion
      revision: kernelci.org
```
The `staging.kernelci.org` branch should be set with a patch in [`kernelci-deploy`](https://github.com/kernelci/kernelci-deploy).

Reply:
OK, just updated with this change.